llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -22514,14 +22514,22 @@ static SDValue performSTNT1Combine(SDNode *N, SelectionDAG &DAG) {
 ///   movi v0.2d, #0
 ///   str q0, [x0]
 ///
-static SDValue replaceZeroVectorStore(SelectionDAG &DAG, StoreSDNode &St) {
+static SDValue replaceZeroVectorStore(SelectionDAG &DAG, StoreSDNode &St,
+                                      AArch64Subtarget const &Subtarget) {
```

Comment:
nit: this form is more common in LLVM:
```suggestion
                                      const AArch64Subtarget &Subtarget) {
```